sklearnex/preview/covariance/covariance.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+from scipy import sparse as sp
+from sklearn.covariance import EmpiricalCovariance as sklearn_EmpiricalCovariance
+from sklearn.utils import check_array
+
+from daal4py.sklearn._utils import sklearn_check_version
+from onedal.common.hyperparameters import get_hyperparameters
+from onedal.covariance import EmpiricalCovariance as onedal_EmpiricalCovariance
+
+from ..._device_offload import dispatch
+from ..._utils import PatchingConditionsChain, register_hyperparameters
+
+
+@register_hyperparameters({"fit": get_hyperparameters("covariance", "compute")})
+class EmpiricalCovariance(sklearn_EmpiricalCovariance):
```

Comment:
All patched estimators are required to have `n_jobs` control and hyperparameters (if applicable) decorators

Reply:
Are n_jobs required for Covariance?

Reply:
https://github.com/intel/scikit-learn-intelex/blob/61f27db25b62d74a53fb46ec040121670bd3d21f/examples/sklearnex/n_jobs.py#L17-L18

https://github.com/intel/scikit-learn-intelex/blob/61f27db25b62d74a53fb46ec040121670bd3d21f/examples/sklearnex/n_jobs.py#L42-L43

https://github.com/intel/scikit-learn-intelex/blob/61f27db25b62d74a53fb46ec040121670bd3d21f/examples/sklearnex/n_jobs.py#L55-L56

Reply:
Thnx for links.
https://github.com/intel/scikit-learn-intelex/pull/1480/files#r1426643617
We are failing some tests in this case. Are we going to fix it?